### PR TITLE
Manual: Plugin & Particle Sections, Map

### PR DESCRIPTION
**Pull request by ax3l**
The headlines and section level for plugins was wrong. Lower period syntax and general post-processing infos.

Also cleans up the particle doc structure in Usage.

Also adds user map in HTML version as iframe.

### CODE REVIEW

File: include/picongpu/param/speciesInitialization.param [hunk 1]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/Bremsstrahlung/include/picongpu/param/speciesInitialization.param [hunk 2]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/Bremsstrahlung/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/Bunch/include/picongpu/param/speciesInitialization.param [hunk 3]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/Bunch/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/FoilLCT/include/picongpu/param/speciesInitialization.param [hunk 4]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/FoilLCT/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/KelvinHelmholtz/include/picongpu/param/speciesInitialization.param [hunk 5]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/KelvinHelmholtz/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/LaserWakefield/include/picongpu/param/speciesInitialization.param [hunk 6]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/LaserWakefield/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/SingleParticleTest/include/picongpu/param/speciesInitialization.param [hunk 7]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/SingleParticleTest/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/ThermalTest/include/picongpu/param/speciesInitialization.param [hunk 8]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/ThermalTest/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/WarmCopper/include/picongpu/param/speciesInitialization.param [hunk 9]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/WarmCopper/include/picongpu/param/speciesInitialization.param]: *another*

File: share/picongpu/examples/WeibelTransverse/include/picongpu/param/speciesInitialization.param [hunk 10]
```diff
@@ -20,12 +20,10 @@
...
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```
Comment (n01r) [inline share/picongpu/examples/WeibelTransverse/include/picongpu/param/speciesInitialization.param]: *another*

File: docs/source/usage/param/particles/init.rst [hunk 11]
```diff
@@ -1,7 +1,7 @@
 .. _usage-particles:
```
Comment (n01r) [inline docs/source/usage/param/particles/init.rst]: So should the link then also be `.. usage-params-particles:` to reflect the tree structure?

File: include/picongpu/param/speciesInitialization.param (hunk 1 above)
Comment (n01r) [reply include/picongpu/param/speciesInitialization.param]: Sorry, should have just mentioned this once for all `speciesInitialization.param` :sweat_smile:

Comment (ax3l) [reply include/picongpu/param/speciesInitialization.param]: fixed in all places :)

File: docs/source/usage/param/particles/init.rst (hunk 11 above)
Comment (ax3l) [reply docs/source/usage/param/particles/init.rst]: yep, done now (`usage-params-core-...`)

File: docs/source/usage/param/particles/init.rst [hunk 12]
```diff
@@ -1,151 +1,151 @@
-.. _usage-particles:
+.. _usage-params-core-particles:
 
 Particles
-=========
+"""""""""
```
[the remark this answers is not in the document]
Comment (ax3l) [reply docs/source/usage/param/particles/init.rst]: Because I want them as a sub-section, not a section.

[the remark this answers is not in the document]
Comment (n01r) [reply docs/source/usage/param/particles/init.rst]: @PrometheusPi for reference: [sections](https://devguide.python.org/documenting/#sections)

**Pull request merged**